doc/benchmark/modules/ROOT/pages/run/running-benchmark-ansible.adoc

```diff
@@ -39,12 +39,19 @@ The most important parameters to customize in `env.yml` are:
 
 `cluster_size`:: Number of instances to be created.
 As default, Gatling will create new HTTP connections for each new user.
-As the network stack of the load driver can be congested with a lot of connections in the `TIME_WAIT` state, consider using one EC2 instance per 300 users per seconds.
+As the network stack of the load driver can be congested with a lot of connections in the `TIME_WAIT` state, consider using one EC2 instance per 250 users per seconds.
 
 `instance_type`:: Size of instances to be created, see https://aws.amazon.com/ec2/instance-types/[AWS instance types].
 
 === Creating EC2 instances
 
+NOTE: Instances will be bound to the IP address of the system which creates them!
+
+When creating the instances, the public IP address of the host creating the machines is added to the EC2 security group, and only this IP address is allowed to log in to the load drivers via SSH.
+When the public IP address changes you'll need to re-run the `create` command.
```

Comment:
Is it necessary to delete and then create? Or is it enough to run create?

Reply:
From what I understand and tested, the create is idempotent and will update existing instances of the EC2 security group as needed. So no delete is necessary to refresh the configuration.